site/content/concepts/tenancy.md

```diff
@@ -3,10 +3,134 @@ title = "Multi Tenancy"
 weight = 405
 +++
 
-This page describes how multi tenancy is handled in the Eclipse Hono project.
+Hono has a multi tenancy concept based on the handling of tenants as own *entities*. All functionality of Hono is 
```

Comment:
It would be great if you would first explain what a *tenant* actually is supposed to be and what it is used for.